README.md

```diff
@@ -0,0 +1,11 @@
+# Matcha-dark-sea on Flathub
```

Comment:
No need to have documentation here. Flatpak will auto-install matching theme from Flathub if it's available.